nixos/modules/services/networking/minidlna.nix

```diff
@@ -106,29 +88,17 @@ in
     };
   };
 
-  imports = [
```

Comment:
no reason to remove these yet, is there? we have instances much older than this

Reply:
I presume this module is not widely used anyway. Let's clean it up and hopefully not touch it again.